person.rb

```diff
@@ -1,5 +1,7 @@
-class Person
+require_relative 'nameable'
+class Person < Nameable
   def initialize(age, name = 'unknown', parent_permission: true)
+    super()
     @id = Random.rand(1..188)
     @name = name
     @age = age
```

Comment:
- [ ] After you imported the `nameable` using `require_relative`, it's important to add the `correct_name` method here as well to return the passed `name`. Please add the method correctly in the `Person` class. 🙏